README.md

```diff
@@ -45,4 +51,4 @@ To translate between SPIR 1.2/2.0 and SPIR-V:
 
 * -o file_name - to specify output name
 * -spirv-debug - output debugging information
-* -spirv-text - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
\ No newline at end of file
+* -spirv-text - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
```

Comment:
For those wondering what the change is on this line: it simply adds a newline at the end of the file.